integration_test/Bindings/Tcl/simple.tcl

```diff
@@ -0,0 +1,36 @@
+# REQUIRES: bindings_tcl
+# RUN: tclsh %s -- %TCL_PATH%
```

Comment:
I think you'll need to pipe this through `FileCheck` for the below `CHECK` lines to actually run.